content/en/tracing/app_analytics/_index.md

```diff
@@ -306,15 +306,16 @@ tracer.use('mysql', {
 
 Database tracing is not captured by App Analytics by default and you must enable collection manually for each integration. For example, to enable App Analytics for ADO.NET:
 
-* Environment Variable or AppSetting: `DD_ADONET_ANALYTICS_ENABLED=true`
+* Windows: Environment Variable or AppSetting: `DD_ADONET_ANALYTICS_ENABLED=true`
+* Linux: Environment Variable or AppSetting: `DD_AdoNet_ANALYTICS_ENABLED=true`
```

Comment:
Should we just say `DD_AdoNet_ANALYTICS_ENABLED` for both? (on Windows it is case-insensitive)

Reply:
I'm not sure if you can see my latest changes, but does "Linux and Windows" for Line 310 work?

Reply:
We shouldn't have with 309 with "Windows" and 310 with "Linux and Windows". We should have only one line.

Reply:
Got it, thanks for making the proposed change!